### DISCUSSION TITLE
[file_picker]  error:  unexpected element <queries> found in <manifest>

### DISCUSSION THREAD

**Discussion by godgege** [singerdmx/flutter-quill#182] [General]
[file_picker]  error:  unexpected element <queries> found in <manifest>

It is recommended to separate QuillToolbar's pictures and other dependencies,
Separate a new toolkit to avoid more dependency issues.
Sometimes people prefer to use your dependency packages in layers and customize editing tools.

**Comment by godgege** [comment]
My English is not good, thank you for your bag, it has great potential

**Comment by matuella** [comment]
I agree with you, we should make this library contain only the essential and allow full customization, not only because it may cause dependency issues, but also constraint this package's usage, making useless for some full-customized use-cases.

Even with this in mind, I think we could aim to have pre-built packages (like we have right now with `image_picker`, `flutter_colorpicker` and others), but keep the core package without any of these dependencies (this is what the JS `quill` actually does with modules)

**Reply by godgege** [reply]
Oh, I understand what you mean, yes, you have considered it well, it should be developed first, and then these aspects should be considered.
But for developers who have problems, there should be a document of possible problems to help more developers transition problems. Smart developers are used to look at issues first.

The tool level of this package is particularly easy to understand. It is easy to customize the construction of the toolbar and realize the corresponding functions. Thank you for your contribution.

**Reply by godgege** [reply]
To solve this problem:  https://github.com/miguelpruivo/flutter_file_picker/wiki/Troubleshooting#-issue-2

**Reply by singerdmx** [reply]
You can take https://github.com/singerdmx/flutter-quill/pull/183 
I didn't finish. Still need to pass through imagePickImpl

**Reply by godgege** [reply]
Thank you, very efficient, support you,
I changed the gradle version here, and it was resolved.
